### PR TITLE
Locale update for OJS 2.4.8-1 (pkp-lib)

### PR DESCRIPTION
**Pull request by mtub**
pkp/pkp-lib#1560

### CODE REVIEW

File: locale/tr_TR/manager.xml [hunk 1]
```diff
@@ -238,4 +238,18 @@
...
+	<message key="manager.reviewerSearch.form.instructions">Aramak istediğiniz terimler için maksimum değerleri aşağıdaki forma giriniz. Form, o alanlardaki verilerin hesaplanmış ortalamaları ile ön tamamlamalıdır.</message>
+	<message key="plugins.categories.citationLookup.description">Atıf Veri Tabanı Bağlama Eklentisi atıf bakman için atıf veri tabanına bağlanır. Ör.: Atıf düzeltirken.</message>
+	<message key="plugins.categories.citationOutput.description">Atıf Çıktı Eklentisi atıf standartlarını tamamlar. Bu Atıf Biçim Eklentisinin yerini alır.</message>
+	<message key="manager.setup.citationFilterParserDescription"><![CDATA[<p>To find and check citations in citation databases they first have to be split into machine readable metadata: title, publisher, publication date, etc. must all be extracted from the original citation text provided by the author. Connectors to external web services can assist in this process. One or several connectors can be added by clicking on the "Add Item" button below. Some of these connectors may need to be configured; you will be prompted if this is the case.</p>
```
Comment (asmecher) [inline locale/tr_TR/manager.xml]: Added English (sorry, I suppose I should've caught this on the `ojs-dev-2_4` branch)

File: templates/email/email.tpl [hunk 2]
```diff
@@ -57,15 +57,15 @@ function deleteAttachment(fileId) {
...
+	<br />
 {/if}
 
 <table class="data" width="100%">
 <tr valign="top">
 	<td class="label" width="20%">{fieldLabel name="to" key="email.to"}</td>
 	<td width="80%" class="value">
 		{foreach from=$to item=toAddress}
-			<input type="text" name="to[]" id="to" value="{if $toAddress.name != ''}{$toAddress.name|escape} &lt;{$toAddress.email|escape}&gt;{else}{$toAddress.email|escape}{/if}" {if !$addressFieldsEnabled}disabled="disabled" {/if}size="40" maxlength="120" class="textField" /><br/>
+			<input type="text" name="to[]" id="to" value="{if $toAddress.name != ''}{$toAddress.name|escape} &lt;{$toAddress.email|escape}&gt;{else}{$toAddress.email|escape}{/if}" {if !$addressFieldsEnabled}disabled="disabled" {/if}size="40" maxlength="120" class="textField" /><br />
```
Comment (asmecher) [inline templates/email/email.tpl]: Accidentally cherry-picked code change?

Comment (mtub) [inline templates/email/email.tpl]: No, just the cherry-picked commit with the <br/> -> <br /> correction…

File: locale/tr_TR/manager.xml (hunk 1 above)
Comment (mtub) [inline locale/tr_TR/manager.xml]: Will update soon.

**Comment by mtub** [comment]
Give me a minute here, will run additional checks and do another commit…

**Comment by mtub** [comment]
Ok, @asmecher , sorry for the delay. Did run another test, removed some English text, etc. - Not so much has changed, and I've also checked the line endings. Ready to merge from my perspective. I can cherry-pick and add to the open `ojs-dev-2_4` @ pkp-lib PR after you've signed off on this.

File: templates/email/email.tpl (hunk 2 above)
Comment (asmecher) [inline templates/email/email.tpl]: Ah! I mis-recognized this as another change. You're right.

**Comment by asmecher** [comment]
Excellent! Huge thanks, @mtub.

**Comment by mtub** [comment]
I can cherry-pick today's updates to this PR to the ojs-dev-2_4 branch. Please let me know who of us is going to do it, @asmecher  `#toomanybranches`

**Comment by asmecher** [comment]
If you can list them off, I'll happily cherry-pick them!

**Comment by mtub** [comment]
I can cherry-pick today's updates to this PR to the ojs-dev-2_4 branch. Please let me know who of us is going to do it, @asmecher  `#toomanybranches`

**Pull request merged**